GVFS/GVFS.Common/DiskLayoutUpgrade.cs

```diff
@@ -73,33 +73,33 @@ public static bool TryCheckDiskLayoutVersion(ITracer tracer, string enlistmentRo
             {
                 if (TryGetDiskLayoutVersion(tracer, enlistmentRoot, out majorVersion, out minorVersion, out error))

```

Comment:
Could you add a functional test that validates that `gvfs mount` will fail when trying to mount a repo on Mac that's version 17?  (see `DiskLayoutUpgradeTests` for examples of how we test this on Windows)

Reply:
DiskLayoutVersionTests.cs added.  This swaps version based on Mac/Windows running on both.